WMF Framework/WMFCaptchaViewController.swift

```diff
@@ -4,6 +4,23 @@ import UIKit
 // Presently it is assumed this view controller will be used only as a
 // child view controller of another view controller.
 
+extension UIView {
+    fileprivate func wmf_subviews(hide: Bool) {
```

Comment:
naming here is a little confusing - could it be a `var wmf_subviewsHidden: Bool` with set & get implementations?